docs/store/llm-artifacts.md

```diff
@@ -0,0 +1,134 @@
+(llm-prompt-artifacts)=
+# LLM prompt artifacts
+
+LLM prompt artifacts capture a prompt definition for large language model (LLM) interactions. 
+
+**In this section**
+- [SDK](#sdk)
+- [LLM prompt artifacts](#llm-prompt-artifacts)
+- [Log LLM prompt artifacts](#log-llm-prompt-artifacts)
+- [Deleting prompt artifacts](#deleting-prompt-artifacts-using-the-sdk)
+- [Viewing LLM-prompt artifacts in the UI](#viewing-llm-prompt-artifacts-in-the-ui)
+
+## SDK
+- {py:class}`~mlrun.projects.MlrunProject.log_llm_prompt`: Logs an LLM prompt artifact to the current project.
+- {py:class}`~mlrun.projects.MlrunProject.list_llm_prompts`: Lists LLM prompt artifacts in the current project with support for filtering.
+- {py:class}`~mlrun.projects.MlrunProject.paginated_list_llm_prompts`: Retrieves a paginated list of LLM prompt artifacts in the current project.
+
+## LLM prompt artifacts
+
+The **prompt template** format is a rformat will be a list[dict], typically with two roles: system and user.
```

Comment:
small typo - `rformat` 
i don't think we should be specific here - `typically with two roles: system and user`, we can write that it will support any role the model support like: system and user

Reply:
updated